dockerfiles/Dockerfile

```diff
@@ -29,8 +29,8 @@ RUN apk --no-cache upgrade && \
 	chmod 0700 /root/.ssh && \
 	passwd -u root && \
 	# install dependencies of conduct2 used by perf
-	pip2 install filelock twisted requests queuelib ujson psutil crochet msgpack-python unidecode attrdict service_identity && \
-	pip2 install git+https://github.com/esnme/ultrajson.git
+	pip2 install filelock twisted requests queuelib psutil crochet msgpack-python unidecode attrdict service_identity && \
+	pip2 install git+https://github.com/esnme/ultrajson.git@v1.35
```

Comment:
The `conductor2` depends on `ujson < 2.0`, so we need to install it in this way.